src/window.rs

```diff
@@ -380,6 +395,45 @@ impl Window {
     pub fn set_decorations(&self, decorations: bool) {
         self.window.set_decorations(decorations)
     }
+    
+    /// Centers the window on the current monitor the window lives in.
+    /// 
+    /// ## Platform-specific
+    /// 
+    /// This only works on Windows and X11.
+    /// It's not tested on macOS.
+    /// 
+    /// This has no effect on Android, Wayland or iOS.
+    pub fn center(&self) {
```

Comment:
Why is this needed in addition to `set_centered`? The naming of state changing functions is already inconsistent (see also #728). I think having multiple ways to do the same thing is probably not a great idea, especially when getting the default monitor is already really easy.